apps/nextra/pages/en/build/guides/your-first-nft.mdx

```diff
@@ -1,768 +1,453 @@
+- **Node.js**: Make sure you have Node.js (version 16 or later) and npm installed.
+- **Aptos CLI (Optional)**: Having the [Aptos CLI](https://aptos.dev/cli-tools/aptos-cli-tool/install-aptos-cli) can be helpful if you want to inspect on-chain data directly, but it's not required for this tutorial.
 
-```python filename="example.py"
-rest_client = RestClient(NODE_URL)
-faucet_client = FaucetClient(FAUCET_URL, rest_client)
-```
+### Set up Your Project
 
-Using the API client we can create a `TokenClient` that we use for common token operations such as creating collections and tokens, transferring them, claiming them, and so on.
+Create a new directory for your project and initialize a Node.js project:
 
-```python filename="example.py"
-token_client = AptosTokenClient(rest_client)
+```bash filename="Terminal"
+mkdir aptos-digital-asset-tutorial
+cd aptos-digital-asset-tutorial
+npm init -y
 ```
 
-[`common.py`](https://github.com/aptos-labs/aptos-python-sdk/blob/main/examples/common.py) initializes these values as follows:
+This will create a `package.json` file, allowing you to install dependencies and run scripts.
 
-```python filename="example.py"
-NODE_URL = os.getenv("APTOS_NODE_URL", "https://api.devnet.aptoslabs.com/v1")
-FAUCET_URL = os.getenv(
-    "APTOS_FAUCET_URL",
-    "https://faucet.devnet.aptoslabs.com",
-)
-```
+### Install Dependencies
 
-<Callout type="info">
-
-By default, the URLs for both the services point to Aptos devnet services. However, they can be configured with the following environment variables:
+You will need the Aptos TypeScript SDK and `dotenv` to manage environment variables:
 
-- `APTOS_NODE_URL`
-- `APTOS_FAUCET_URL`
-</Callout>
-  </Tabs.Tab>
-  </Tabs>
-
----
+```bash filename="Terminal"
+npm install @aptos-labs/ts-sdk dotenv
+npm install --save-dev @types/node
+```
 
-### Step 4.2: Creating local accounts
+### Create tsconfig.json
 
-The next step is to create two accounts locally. [Accounts](../../network/blockchain/accounts.mdx) consist of a public address and the public/private key pair used to authenticate ownership of the account. This step demonstrates how to generate an Account and store its key pair and address in a variable.
+1. Create a new TypeScript configuration file:
 
-<Tabs items={["TypeScript", "Python"]}>
+<Tabs items={["Mac/Linux", "Windows"]}>
   <Tabs.Tab>
-
-```ts filename="example.ts"
-const alice = Account.generate();
-const bob = Account.generate();
-```
-
+    ```bash filename="Terminal"
+    touch tsconfig.json
+    ```
   </Tabs.Tab>
   <Tabs.Tab>
-
-```python filename="example.py"
-alice = Account.generate()
-bob = Account.generate()
-```
-
+    ```bash filename="Terminal"
+    type nul > tsconfig.json
+    ```
   </Tabs.Tab>
 </Tabs>
 
-<Callout type="info">
-Note that this only generates the local keypair. After generating the keypair and public address, the account still does not exist on-chain.
+2. Open the `tsconfig.json` file and add the following configuration:
```

Comment:
Can we skip this? I think we should assume that the user knows how to make a new file on their machine. Ideally it just looks like "Create a tsconfig file with the following:"